src/constants/pollution.ts

```diff
@@ -0,0 +1,8 @@
+const pollutionLevels = {
+  'PM2.5': [0, 25, 50, 100, 300, Infinity],
+  PM10: [0, 40, 80, 120, 300, Infinity]
+};
+
+export default {
+  pollutionLevels
+};
```

Comment:
```ts
export default {
  pollutionLevels: {
    'PM2.5': [0, 25, 50, 100, 300, Infinity],
    PM10: [0, 40, 80, 120, 300, Infinity]
  }
};
```

It's just my personal preference. To be consistent with other files with constants (e.g. `colors` and `pahts`, you can write everting inside the exported object without creating helper variables). But that's only my preference.